Java/main-sample/src/main/java/com/perfectomobile/appTest/MyApp.java

```diff
@@ -99,25 +97,29 @@ public static void main(String[] args) throws IOException {
             //step1: Validate login page
             reportiumClient.stepStart("step1: Validate login page");
             WebDriverWait wait = new WebDriverWait(driver, 15);  // timeout of 15 seconds
-            try {
-                wait.until(ExpectedConditions.visibilityOf(driver.findElementByXPath("//*[@resource-id='com.bloomfire.android.perfecto:id/sso']")));
-            } catch (TimeoutException t) {
-                System.out.println("Did not find the Label within explicit wait time");
-            }
+            //try {
+            //    wait.until(ExpectedConditions.visibilityOf(driver.findElementByXPath("//*[@resource-id='com.bloomfire.android.perfecto:id/sso']")));
+            //} catch (TimeoutException t) {
+            //    testpassed=false;
```

Comment:
?

Reply:
some selenuim exception came up here, we ignore failures here anyway so we can rever this if it looks better

Reply:
yes comments look weird